.travis.yml

```diff
@@ -16,5 +18,8 @@ matrix:
   include:
     - env: BUILD_SYSTEM=qmake
     - env: BUILD_SYSTEM=cmake
+    - env: BUILD_VARIANT=click
+      script:
+        - clickable clean build click-build
```

Comment:
Even if this is only one command, I'd prefer to add `clickable clean build click-build` to `travis/build.sh`. It makes it cleaner, when *every* target uses the `build.sh`.